Home_work6/Home_work6.cpp

```diff
@@ -6,11 +6,11 @@ int main()
     std::cout << "Game start\n";
     while (true) {
         std::cout << "Ultimate quess quest\nEnter -1 for exit\nEnter -2 for Treasure hunter\nTry to guess number (0-50)\n";
-        int highscore = 0;
+        int highscore = -1;
```

Comment:
My bad) It should be INT_MAX as the max possible value, not least possible value)